server/controllers/notifications.js

```diff
@@ -0,0 +1,63 @@
+const Notification = require("../models/Notification");
+const Socket = require("../services/socket");
+
+/*
+NOTIFICATION CODES:
+1 - you have been assigned to do a review
+2 - your request has been matched with someone
+3 - your reviewer has accepted your request
+4 - your reviewer has declined your request
+5 - your reviewer has completed your request
+6 - your requestor has made a new post in the thread
+7 - your reviewer has made a new post in the thread
+*/
```

Comment:
you could turn these codes into constants, like:
`const REVIEWER_ACCEPTED_REQUEST = 1`, and then use the constant to make it a bit more readable

Reply:
added constants, and changed some of the codes to reflect our discussion today